torrent/torrent.go

```diff
@@ -129,7 +129,8 @@ func GenerateMetadataInfo(fqfn string) (*MetadataInfo, error) {
 
 	// If we had a cache, break it into the hashes.
 	if use_cache {
-		hashes := make([][]byte, 0, hashCount)
+		hashes = make([][]byte, 0, hashCount)
+		fmt.Printf("Hash count = %d\n", hashCount)
```

Comment:
Sorry, in a hurry, missed that debug statement. All the others were LogInfos, so I got rid of them by search.